doi/src/intTest/java/ca/nrc/cadc/doi/TestUtil.java

```diff
@@ -96,22 +95,35 @@ public class TestUtil {
     // AUTH_CERT has read/write access to the test DOI
     static String AUTH_CERT = "doi-auth.pem";
 
+    // AUTH_TWO_CERT is part of test publisher group which can approve or reject a DOI if alternative permissions is configured
+    static String AUTH_TWO_CERT = "cadcauthtest2.pem";
```

Comment:
Change this to PUBLISHER_CERT = "doi-publisher.pem"
Then we can create a symlink in $A/test-certificates from doi-publisher.pem -> cadcauthtest2.pem